supportedBackends/google/pipelines/v2alpha1/src/main/resources/transfer.sh

```diff
@@ -0,0 +1,168 @@
+}
+
+localize_directory() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  # Do not quote rpflag, when that is set it will be -u project which should be two distinct arguments.
+  mkdir -p "${container}" && rm -f "$HOME/.config/gcloud/gce" && gsutil ${rpflag} -m rsync -r "${cloud}" "${container}" > "$gsutil_log" 2>&1
+}
+
+# Content type is sometimes (not always) specified for delocalizations.
+gsutil_content_flag() {
+  local content="$1"
+
+  if [[ ! -z "${content}" ]]; then
+    echo -n -- "-h 'Content-Type: $content'"
+  fi
+}
+
+delocalize_file() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  local content="$4"
+
+  local content_flag=$(gsutil_content_flag $content)
+  # Do not quote rpflag or content_flag, when those are set they will be two distinct arguments each.
+  rm -f "$HOME/.config/gcloud/gce" && gsutil ${rpflag} -m ${content_flag} cp "$container" "$cloud" > "$gsutil_log" 2>&1
+}
+
+delocalize_directory() {
+  local cloud="$1"
+  local container="$2"
+  local rpflag="$3"
+  local content="$4"
+
+  local content_flag=$(gsutil_content_flag $content)
+  # Do not quote rpflag or content_flag, when those are set they will be two distinct arguments each.
+  rm -f "$HOME/.config/gcloud/gce" && gsutil ${rpflag} -m ${content_flag} rsync -r "$container" "$cloud" > "$gsutil_log" 2>&1
+}
+
+timestamped_message() {
+  printf '%s %s\n' "$(date -u '+%Y/%m/%d %H:%M:%S')" "$1"
+}
+
+localize_message() {
+  local cloud="$1"
+  local container="$2"
+  local message=$(printf "Localizing input %s -> %s" "$cloud" "$container")
+  timestamped_message "${message}"
+}
+
+delocalize_message() {
+  local cloud="$1"
+  local container="$2"
+  local message=$(printf "Delocalizing output %s -> %s" "$container" "$cloud")
+  timestamped_message "${message}"
+}
+
+# Transfer a bundle of files xor directories to or from the same GCS bucket.
+transfer() {
+  local direction="$1"
+  local files_or_directories="$2"
+  local project="$3"
+  local max_attempts="$4"
+
+  shift; shift; shift; shift # direction; files_or_directories; project; max_attempts
+
+  if [[ "$direction" != "localize" && "$direction" != "delocalize" ]]; then
+    echo "direction must be 'localize' or 'delocalize' but got '$direction'"
+    exit 1
+  fi
+
+  transfer_fn_name=""
+  if [[ "$files_or_directories" = "files" ]]; then
+    transfer_fn_name="${direction}_file"
+  elif [[ "$files_or_directories" = "directories" ]]; then
+    transfer_fn_name="${direction}_directory"
+  else
+    echo "files_or_directories must be 'files' or 'directories' but got '$files_or_directories'"
+    exit 1
+  fi
+
+  # Whether the requester pays status of the GCS bucket is certain. rp status is presumed false until proven otherwise.
+  local rp_status_certain=false
+  local use_requester_pays=false
+
+  local message_fn="${direction}_message"
+
+  # One race-condition sidestepping sleep 5 to rule them all.
+  sleep 5
```

Comment:
You should be able to remove this now.  We have a work around in our worker for this docker bug.